spec/features/index_page_spec.rb

```diff
@@ -22,6 +22,15 @@
     expect(page).to have_content(customer.email)
   end
 
+  it "links to the customer show page without javascript", js: false do
```

Comment:
Oh sorry... re-reading your comment. Do you mean invert the test names? So `links to the customer show page` is the vanilla version, then `links to the customer show page with javascript` or similar?